sources/cli.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+func consumeArgument(flag: String, from arguments: inout ArraySlice<String>) -> String? {
```

Comment:
Your call, but `flag` implies `Bool` in my mind. `name` might be more descriptive?

Reply:
done